### PR TITLE
Issue 13 add update tray icon

### PR DESCRIPTION
**Pull request by VsevolodGolovanov**

### CODE REVIEW

File: src/main/java/org/cryptomator/integrations/tray/TrayMenuController.java [hunk 1]
```diff
@@ -30,6 +30,13 @@ static Optional<TrayMenuController> get() {
 	 */
 	void showTrayIcon(byte[] imageData, Runnable defaultAction, String tooltip) throws TrayMenuException;
 
+	/**
+	 * Updates an icon on the system tray.
```
Comment (overheadhunter) [inline src/main/java/org/cryptomator/integrations/tray/TrayMenuController.java:34]:
```suggestion
	 * Updates the icon on the system tray.
```

We don't want to update any arbitrary tray icon. 😉

**Review by overheadhunter — CHANGES_REQUESTED** [review]
Please also specify that this method throws an IllegalStateException, when it is called before an icon has been added.

**Comment by VsevolodGolovanov** [comment]
@overheadhunter, done.

**Review by overheadhunter — APPROVED** [review]

File: src/main/java/org/cryptomator/integrations/tray/TrayMenuController.java (hunk 1 above)
Comment (infeo) [reply src/main/java/org/cryptomator/integrations/tray/TrayMenuController.java]: Done in https://github.com/cryptomator/integrations-api/pull/14/commits/c7ecab8105edad15f4fde22b343282a6c31d2e12

**Comment by infeo** [comment]
Thanks @VsevolodGolovanov. We will release a new version of the integrations-api lib very soon!

**Pull request merged**